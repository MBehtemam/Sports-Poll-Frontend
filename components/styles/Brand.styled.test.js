import React from "react";
import styled from "styled-components";
import renderer from "react-test-renderer";
import "jest-styled-components";

import BrandStyled from "./Brand.styled";

describe("Test suits for brand", () => {
  it("should match with snapshot", () => {
    const tree = renderer.create(<BrandStyled />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("should has correct style rules", () => {
    const tree = renderer.create(<BrandStyled />).toJSON();
    expect(tree).toHaveStyleRule("flex", "1");
  });
});
