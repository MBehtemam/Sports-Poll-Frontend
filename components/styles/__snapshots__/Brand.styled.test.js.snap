// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for brand should match with snapshot 1`] = `
.c0 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  padding-left: 20px;
}

<div
  className="c0"
/>
`;
